modules/ROOT/pages/product-versioning.adoc

```diff
@@ -21,13 +21,50 @@ A technical Id also exists for Bonita components, and it follows the usual seman
 
 For example, upgrading from a 2021.1-0205 to a 2021.1-0401 will not require any migration. However, upgrades from a 7.11.5 to a 2021.1-0205 will require a migration.
 
-Technical Id Bonita version correspondence table
+Here is a Bonita Platform version / Technical Id / UI Designer version correspondence table:
 
 |===
-| Bonita Version | Technical Id
-
+| Release date | Bonita Platform version | Technical Id | UI Designer version
```

Comment:
Are you sure you want to put release dates here? This adds maintenance tasks.
They are often missing in the release notes (see #1853 for instance), so I guess this will be the case here too.

Anyway, as this page must now be updated on new releases as the release-notes page, I guess we can enforce it is updated as part of the Release process, so adding the date is not a big deal.

Reply:
Thanks for this feedback.
We will need to add the release numbers for each release, so I think the date is not that much to add on the top of it.
Yes, the newly-arrived POs are now aware that the date is needed for each release in the release notes as well.
And yes, this will be part of Serenity.